services/web/server/src/simcore_service_webserver/projects/_nodes_api.py

```diff
@@ -135,33 +246,24 @@ async def fake_screenshots_factory(
             )
 
     elif node.key.startswith("simcore/services/dynamic"):
-        # For dynamic services, just create fake images
-
-        # References:
-        # - https://github.com/Ybalrid/Ogre_glTF
-        # - https://placehold.co/
-        # - https://picsum.photos/
-        #
-        count = int(str(node_id.int)[0])
-        text = urllib.parse.quote(node.label)
-
-        screenshots = [
-            *(
-                NodeScreenshot(
-                    thumbnail_url=f"https://picsum.photos/seed/{node_id.int + n}/170/120",  # type: ignore[arg-type]
-                    file_url=f"https://picsum.photos/seed/{node_id.int + n}/500",  # type: ignore[arg-type]
-                    mimetype="image/jpeg",
-                )
-                for n in range(count)
-            ),
-            *(
-                NodeScreenshot(
-                    thumbnail_url=f"https://placehold.co/170x120?text={text}",  # type: ignore[arg-type]
-                    file_url=f"https://placehold.co/500x500?text={text}",  # type: ignore[arg-type]
-                    mimetype="image/svg+xml",
-                )
-                for n in range(count)
-            ),
-        ]
+        # when dealing with dynamic service scan the assets directory and
+        # pull in all the assets that have been dropped in there
+
+        assets_files: list[FileMetaDataGet] = await get_files_in_node_folder(
+            app=request.app,
+            user_id=user_id,
+            project_id=project_id,
+            node_id=node_id,
+            folder_name=ASSETS_FOLDER,
+        )
+
+        resolved_screenshots: list[
+            NodeScreenshot
+        ] = await _get_node_screenshots_parallel(
```

Comment:
what means _get_node_screenshots_parallel ?

```suggestion
        ] = await _get_node_screenshots(
```